packages/rrweb-worker/src/index.ts

```diff
@@ -0,0 +1,9 @@
+import workerString from './image-bitmap-data-url-worker';
+
+/**
+ * Get the URL for a web worker.
+ */
+export function getImageBitmapDataUrlWorkerURL(): string {
+  const workerBlob = new Blob([workerString]);
+  return URL.createObjectURL(workerBlob);
```

Comment:
Does this mean we'll have potential CSP errors now?

Reply:
we already had before! Previously this was just being rewritten to a string-based worker by rollup under the hood 😬